centreon/www/api/class/webService.class.php

```
@@ -67,7 +67,7 @@ public function __construct()
      * @param bool $isInternal If the api is call in internal
      * @return bool If the user has access to the action
      */
-    public function authorize($action, $user, $isInternal = false)
+    public function authorize($action, $user, $isInternal = false): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$action'. (undefined)

(UnusedFormalParameter)